mobx_codegen/lib/src/mobx_codegen_base.dart

```diff
@@ -68,7 +68,7 @@ class StoreGenerator extends Generator {
       });
 
       yield _generateCodeFromTemplate(
-          mixedClass.name, baseClass, MixinStoreTemplate(), typeNameFinder);
+          mixedClass.name3!, baseClass, MixinStoreTemplate(), typeNameFinder);
```

Comment:
The new `name3` from the Element2 API is nullable. This is inherited from the base `Element2`. In a number of places, I have made the assumption that the name in effect cannot be null, as in the case of this `ClassElement2`. This may be less safe than I think, or I may have made a mistake. I'm happy to change this approach.